parser.go

```diff
@@ -524,15 +529,23 @@ func (p *parser) parseEntryPoint() error {
 	return nil
 }
 
-func parseTags(comment string) (*TagDefinition, error) {
+func parseTags(comment string, tagGroups map[string][]string, tagGroupKeys *[]string) (*TagDefinition, error) {
 	re := regexp.MustCompile("\"([^\"]*)\"")
 	matches := re.FindAllStringSubmatch(comment, -1)
-	if len(matches) == 0 || len(matches[0]) == 1 {
-		return nil, fmt.Errorf("Expected: @Tags \"<name>\" [\"<description>\"] Received: %s", comment)
+	if len(matches) != 3 || len(matches[0]) == 1 {
```

Comment:
Why this change? Why is 3 the number we are checking?

Reply:
the 3 count comes from "group(1)" "name(2)" "description(3)", if we have less or more then there isn't the right number for parsing and we want to early exit

Reply:
Can we add a comment with where the 3 count comes from? i.e your "group(1)" "name(2)" "description(3)" would work perfectly imo

Reply:
I decided to switch over to a method comment with more detail. I want to be careful about over-commenting on code that should be closer to self documenting.

Completely agree this code is due for a refactor or clean up, but not in scope for this change.